Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

error: implicit declaration of function 'pypynotify_register_classes' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

error: implicit declaration of function 'pypynotify_add_constants' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2006-10-08 13:04:35.000000000 -0500
+++ configure	2020-12-17 08:45:55.000000000 -0600
@@ -4242,8 +4242,8 @@
   for (i = 0; i < 256; i++)
     if (XOR (islower (i), ISLOWER (i))
 	|| toupper (i) != TOUPPER (i))
-      exit(2);
-  exit (0);
+      return 2;
+  return 0;
 }
 _ACEOF
 rm -f conftest$ac_exeext
--- src/pynotifymodule.c.orig	2020-12-17 08:47:20.000000000 -0600
+++ src/pynotifymodule.c	2020-12-17 08:50:26.000000000 -0600
@@ -3,7 +3,8 @@
 #include <pygtk/pygtk.h>
 #include <libnotify/notify.h>
 
-void pynotify_register_classes(PyObject *d);
+void pypynotify_add_constants(PyObject *module, const gchar *strip_prefix);
+void pypynotify_register_classes(PyObject *d);
 extern PyMethodDef pypynotify_functions[];
 
 DL_EXPORT(void)
